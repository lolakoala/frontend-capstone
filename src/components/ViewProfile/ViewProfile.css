.profile-pic {
  height: 200px;
  width: auto;
  margin-left: 30px;
  margin-right: 30px;
  display: inline-block;
  border: 5px solid #315659;
  padding: 5px;
}

.pic-name-city {
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  margin-bottom: 50px;
}

.name-city {
  color: #315659;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
}

.name-city h3 {
  font-size: 36px;
  font-weight: bold;
  margin-top: 0;
  margin-bottom: 10px;
}

.name-city h4 {
  font-size: 28px;
  font-weight: lighter;
  margin-top: 0px;
  margin-bottom: 0px;
}

.name-city p {
  display: inline-block;
  height: 120px;
  overflow: scroll;
}

.profile-bottom {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: flex-start;
  margin-bottom: 100px;
}
