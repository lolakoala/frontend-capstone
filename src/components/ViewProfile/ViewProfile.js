import React from 'react';
import PropTypes from 'prop-types';
import icon from '../../assets/snail-shell.svg';
import css from './ViewProfile.css';
import { List,
  ListItem,
  Subheader,
  RaisedButton
} from 'material-ui';

const ViewProfile = ({ currentUser, userChallenges, history }) => {

  // const userChallenges = ['depression', 'anxiety'];
  // const currentUser = {
  //   city: 'Denver',
  //   userName: 'lolakoala',
  //   aboutMe: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,
  // sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
  // Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
  // ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
  // in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
  // sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
  // };

  const mapChallenges = userChallenges.map((challenge, index) => {
    return <ListItem
      primaryText={challenge}
      leftAvatar={<img src={icon} alt="snail shell icon" height="40" width="40" />}
      style={{
        fontSize: '18px',
        fontWeight: 'bold',
        color:'#2F9C95'
      }}
      disabled={true}
      key={`${challenge}${index}`}
    />;
  });

  const editProfile = () => {
    history.push('/editProfile');
  };


  return (
    <div>
      <div className='pic-name-city'>
        {/* <img
          // placeholder img url
          src="https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg"
          alt="Your avatar"
          className="profile-pic"
        /> */}
        <div className='name-city'>
          <h3>{currentUser.userName}</h3>
          <h4>{currentUser.city}</h4>
          <p>{currentUser.aboutMe}</p>
        </div>
      </div>
      <div className='profile-bottom'>
        <List>
          <Subheader style={{
            fontSize: '30px',
            fontWeight: 'bold',
            color: '#253031'
          }}>Your Challenges</Subheader>
          {mapChallenges}
        </List>
        <RaisedButton
          label="Edit Profile"
          onClick={editProfile}
        />
      </div>
    </div>
  );

};

export default ViewProfile;

ViewProfile.propTypes = {
  currentUser: PropTypes.object,
  userChallenges: PropTypes.array,
  history: PropTypes.object
};
