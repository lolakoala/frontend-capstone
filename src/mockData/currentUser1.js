const currentUser1 = {
  email: 'lola@gmail.com',
  city: 'Denver',
  userName: 'lolakoala',
  aboutMe: 'Lola loves doggos.'
};

export default currentUser1;
