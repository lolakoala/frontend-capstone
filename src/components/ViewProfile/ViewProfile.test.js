import React from "react";
import ViewProfile from './ViewProfile.js';
import { shallow, configure } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import currentUser1 from '../../mockData/currentUser1.js';

configure({ adapter: new Adapter() });

describe('ViewProfile', () => {
  const wrapper = shallow(<ViewProfile
    currentUser={currentUser1}
    userChallenges={['anxiety']}
    history={{}}
  />);

  it('should exist', () => {
    expect(wrapper).toBeDefined();
  });

  it('should match snap', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should have username', () => {
    const userName = wrapper.find('h3');

    expect(userName.length).toEqual(1);
  });

  it('should have city', () => {
    const city = wrapper.find('h4');

    expect(city.length).toEqual(1);
  });

  it('should have about me', () => {
    const aboutMe = wrapper.find('p');

    expect(aboutMe.length).toEqual(1);
  });

  it('should have list', () => {
    const list = wrapper.find('List');

    expect(list.length).toEqual(1);
  });

  it('should have subheader', () => {
    const sub = wrapper.find('Subheader');

    expect(sub.length).toEqual(1);
  });

  it('should have num of list items equal to user challenges', () => {
    const items = wrapper.find('ListItem');

    expect(items.length).toEqual(1);

    it('should have button', () => {
      const button = wrapper.find('RaisedButton');

      expect(button.length).toEqual(1);
    });
  });
});
