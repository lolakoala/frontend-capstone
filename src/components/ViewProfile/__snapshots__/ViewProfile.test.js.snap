// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewProfile should match snap 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ViewProfile
    currentUser={
        Object {
            "aboutMe": "Lola loves doggos.",
            "city": "Denver",
            "email": "lola@gmail.com",
            "userName": "lolakoala",
          }
    }
    history={Object {}}
    userChallenges={
        Array [
            "anxiety",
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="pic-name-city"
>
          <img
                    alt="Your avatar"
                    className="profile-pic"
                    src="https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg"
          />
          <div
                    className="name-city"
          >
                    <h3>
                              lolakoala
                    </h3>
                    <h4>
                              Denver
                    </h4>
                    <p>
                              Lola loves doggos.
                    </p>
          </div>
</div>,
        <div
          className="profile-bottom"
>
          <List>
                    <Subheader
                              inset={false}
                              style={
                                        Object {
                                                  "color": "#253031",
                                                  "fontSize": "30px",
                                                  "fontWeight": "bold",
                                                }
                              }
                    >
                              Your Challenges
                    </Subheader>
                    <ListItem
                              autoGenerateNestedIndicator={true}
                              containerElement="span"
                              disableKeyboardFocus={false}
                              disabled={true}
                              initiallyOpen={false}
                              insetChildren={false}
                              leftAvatar={
                                        <img
                                                  alt="snail shell icon"
                                                  height="40"
                                                  src="snail-shell.svg"
                                                  width="40"
                                        />
                              }
                              nestedItems={Array []}
                              nestedLevel={0}
                              onKeyboardFocus={[Function]}
                              onMouseEnter={[Function]}
                              onMouseLeave={[Function]}
                              onNestedListToggle={[Function]}
                              onTouchEnd={[Function]}
                              onTouchStart={[Function]}
                              open={null}
                              primaryText="anxiety"
                              primaryTogglesNestedList={false}
                              secondaryTextLines={1}
                              style={
                                        Object {
                                                  "color": "#2F9C95",
                                                  "fontSize": "18px",
                                                  "fontWeight": "bold",
                                                }
                              }
                    />
          </List>
          <RaisedButton
                    disabled={false}
                    fullWidth={false}
                    label="Edit Profile"
                    labelPosition="after"
                    onClick={[Function]}
                    primary={false}
                    secondary={false}
          />
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="Your avatar"
              className="profile-pic"
              src="https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg"
/>,
            <div
              className="name-city"
>
              <h3>
                            lolakoala
              </h3>
              <h4>
                            Denver
              </h4>
              <p>
                            Lola loves doggos.
              </p>
</div>,
          ],
          "className": "pic-name-city",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Your avatar",
              "className": "profile-pic",
              "src": "https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  lolakoala
</h3>,
                <h4>
                  Denver
</h4>,
                <p>
                  Lola loves doggos.
</p>,
              ],
              "className": "name-city",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "lolakoala",
                },
                "ref": null,
                "rendered": "lolakoala",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Denver",
                },
                "ref": null,
                "rendered": "Denver",
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Lola loves doggos.",
                },
                "ref": null,
                "rendered": "Lola loves doggos.",
                "type": "p",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <List>
              <Subheader
                            inset={false}
                            style={
                                          Object {
                                                        "color": "#253031",
                                                        "fontSize": "30px",
                                                        "fontWeight": "bold",
                                                      }
                            }
              >
                            Your Challenges
              </Subheader>
              <ListItem
                            autoGenerateNestedIndicator={true}
                            containerElement="span"
                            disableKeyboardFocus={false}
                            disabled={true}
                            initiallyOpen={false}
                            insetChildren={false}
                            leftAvatar={
                                          <img
                                                        alt="snail shell icon"
                                                        height="40"
                                                        src="snail-shell.svg"
                                                        width="40"
                                          />
                            }
                            nestedItems={Array []}
                            nestedLevel={0}
                            onKeyboardFocus={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            onNestedListToggle={[Function]}
                            onTouchEnd={[Function]}
                            onTouchStart={[Function]}
                            open={null}
                            primaryText="anxiety"
                            primaryTogglesNestedList={false}
                            secondaryTextLines={1}
                            style={
                                          Object {
                                                        "color": "#2F9C95",
                                                        "fontSize": "18px",
                                                        "fontWeight": "bold",
                                                      }
                            }
              />
</List>,
            <RaisedButton
              disabled={false}
              fullWidth={false}
              label="Edit Profile"
              labelPosition="after"
              onClick={[Function]}
              primary={false}
              secondary={false}
/>,
          ],
          "className": "profile-bottom",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <Subheader
                  inset={false}
                  style={
                                    Object {
                                                      "color": "#253031",
                                                      "fontSize": "30px",
                                                      "fontWeight": "bold",
                                                    }
                  }
>
                  Your Challenges
</Subheader>,
                Array [
                  <ListItem
                    autoGenerateNestedIndicator={true}
                    containerElement="span"
                    disableKeyboardFocus={false}
                    disabled={true}
                    initiallyOpen={false}
                    insetChildren={false}
                    leftAvatar={
                                        <img
                                                            alt="snail shell icon"
                                                            height="40"
                                                            src="snail-shell.svg"
                                                            width="40"
                                        />
                    }
                    nestedItems={Array []}
                    nestedLevel={0}
                    onKeyboardFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    onNestedListToggle={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                    open={null}
                    primaryText="anxiety"
                    primaryTogglesNestedList={false}
                    secondaryTextLines={1}
                    style={
                                        Object {
                                                            "color": "#2F9C95",
                                                            "fontSize": "18px",
                                                            "fontWeight": "bold",
                                                          }
                    }
/>,
                ],
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "Your Challenges",
                  "inset": false,
                  "style": Object {
                    "color": "#253031",
                    "fontSize": "30px",
                    "fontWeight": "bold",
                  },
                },
                "ref": null,
                "rendered": "Your Challenges",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "anxiety0",
                "nodeType": "class",
                "props": Object {
                  "autoGenerateNestedIndicator": true,
                  "containerElement": "span",
                  "disableKeyboardFocus": false,
                  "disabled": true,
                  "initiallyOpen": false,
                  "insetChildren": false,
                  "leftAvatar": <img
                    alt="snail shell icon"
                    height="40"
                    src="snail-shell.svg"
                    width="40"
/>,
                  "nestedItems": Array [],
                  "nestedLevel": 0,
                  "onKeyboardFocus": [Function],
                  "onMouseEnter": [Function],
                  "onMouseLeave": [Function],
                  "onNestedListToggle": [Function],
                  "onTouchEnd": [Function],
                  "onTouchStart": [Function],
                  "open": null,
                  "primaryText": "anxiety",
                  "primaryTogglesNestedList": false,
                  "secondaryTextLines": 1,
                  "style": Object {
                    "color": "#2F9C95",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "disabled": false,
              "fullWidth": false,
              "label": "Edit Profile",
              "labelPosition": "after",
              "onClick": [Function],
              "primary": false,
              "secondary": false,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="pic-name-city"
>
            <img
                        alt="Your avatar"
                        className="profile-pic"
                        src="https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg"
            />
            <div
                        className="name-city"
            >
                        <h3>
                                    lolakoala
                        </h3>
                        <h4>
                                    Denver
                        </h4>
                        <p>
                                    Lola loves doggos.
                        </p>
            </div>
</div>,
          <div
            className="profile-bottom"
>
            <List>
                        <Subheader
                                    inset={false}
                                    style={
                                                Object {
                                                            "color": "#253031",
                                                            "fontSize": "30px",
                                                            "fontWeight": "bold",
                                                          }
                                    }
                        >
                                    Your Challenges
                        </Subheader>
                        <ListItem
                                    autoGenerateNestedIndicator={true}
                                    containerElement="span"
                                    disableKeyboardFocus={false}
                                    disabled={true}
                                    initiallyOpen={false}
                                    insetChildren={false}
                                    leftAvatar={
                                                <img
                                                            alt="snail shell icon"
                                                            height="40"
                                                            src="snail-shell.svg"
                                                            width="40"
                                                />
                                    }
                                    nestedItems={Array []}
                                    nestedLevel={0}
                                    onKeyboardFocus={[Function]}
                                    onMouseEnter={[Function]}
                                    onMouseLeave={[Function]}
                                    onNestedListToggle={[Function]}
                                    onTouchEnd={[Function]}
                                    onTouchStart={[Function]}
                                    open={null}
                                    primaryText="anxiety"
                                    primaryTogglesNestedList={false}
                                    secondaryTextLines={1}
                                    style={
                                                Object {
                                                            "color": "#2F9C95",
                                                            "fontSize": "18px",
                                                            "fontWeight": "bold",
                                                          }
                                    }
                        />
            </List>
            <RaisedButton
                        disabled={false}
                        fullWidth={false}
                        label="Edit Profile"
                        labelPosition="after"
                        onClick={[Function]}
                        primary={false}
                        secondary={false}
            />
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Your avatar"
                className="profile-pic"
                src="https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg"
/>,
              <div
                className="name-city"
>
                <h3>
                                lolakoala
                </h3>
                <h4>
                                Denver
                </h4>
                <p>
                                Lola loves doggos.
                </p>
</div>,
            ],
            "className": "pic-name-city",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Your avatar",
                "className": "profile-pic",
                "src": "https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3>
                    lolakoala
</h3>,
                  <h4>
                    Denver
</h4>,
                  <p>
                    Lola loves doggos.
</p>,
                ],
                "className": "name-city",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "lolakoala",
                  },
                  "ref": null,
                  "rendered": "lolakoala",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Denver",
                  },
                  "ref": null,
                  "rendered": "Denver",
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Lola loves doggos.",
                  },
                  "ref": null,
                  "rendered": "Lola loves doggos.",
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <List>
                <Subheader
                                inset={false}
                                style={
                                                Object {
                                                                "color": "#253031",
                                                                "fontSize": "30px",
                                                                "fontWeight": "bold",
                                                              }
                                }
                >
                                Your Challenges
                </Subheader>
                <ListItem
                                autoGenerateNestedIndicator={true}
                                containerElement="span"
                                disableKeyboardFocus={false}
                                disabled={true}
                                initiallyOpen={false}
                                insetChildren={false}
                                leftAvatar={
                                                <img
                                                                alt="snail shell icon"
                                                                height="40"
                                                                src="snail-shell.svg"
                                                                width="40"
                                                />
                                }
                                nestedItems={Array []}
                                nestedLevel={0}
                                onKeyboardFocus={[Function]}
                                onMouseEnter={[Function]}
                                onMouseLeave={[Function]}
                                onNestedListToggle={[Function]}
                                onTouchEnd={[Function]}
                                onTouchStart={[Function]}
                                open={null}
                                primaryText="anxiety"
                                primaryTogglesNestedList={false}
                                secondaryTextLines={1}
                                style={
                                                Object {
                                                                "color": "#2F9C95",
                                                                "fontSize": "18px",
                                                                "fontWeight": "bold",
                                                              }
                                }
                />
</List>,
              <RaisedButton
                disabled={false}
                fullWidth={false}
                label="Edit Profile"
                labelPosition="after"
                onClick={[Function]}
                primary={false}
                secondary={false}
/>,
            ],
            "className": "profile-bottom",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <Subheader
                    inset={false}
                    style={
                                        Object {
                                                            "color": "#253031",
                                                            "fontSize": "30px",
                                                            "fontWeight": "bold",
                                                          }
                    }
>
                    Your Challenges
</Subheader>,
                  Array [
                    <ListItem
                      autoGenerateNestedIndicator={true}
                      containerElement="span"
                      disableKeyboardFocus={false}
                      disabled={true}
                      initiallyOpen={false}
                      insetChildren={false}
                      leftAvatar={
                                            <img
                                                                  alt="snail shell icon"
                                                                  height="40"
                                                                  src="snail-shell.svg"
                                                                  width="40"
                                            />
                      }
                      nestedItems={Array []}
                      nestedLevel={0}
                      onKeyboardFocus={[Function]}
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                      onNestedListToggle={[Function]}
                      onTouchEnd={[Function]}
                      onTouchStart={[Function]}
                      open={null}
                      primaryText="anxiety"
                      primaryTogglesNestedList={false}
                      secondaryTextLines={1}
                      style={
                                            Object {
                                                                  "color": "#2F9C95",
                                                                  "fontSize": "18px",
                                                                  "fontWeight": "bold",
                                                                }
                      }
/>,
                  ],
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": "Your Challenges",
                    "inset": false,
                    "style": Object {
                      "color": "#253031",
                      "fontSize": "30px",
                      "fontWeight": "bold",
                    },
                  },
                  "ref": null,
                  "rendered": "Your Challenges",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "anxiety0",
                  "nodeType": "class",
                  "props": Object {
                    "autoGenerateNestedIndicator": true,
                    "containerElement": "span",
                    "disableKeyboardFocus": false,
                    "disabled": true,
                    "initiallyOpen": false,
                    "insetChildren": false,
                    "leftAvatar": <img
                      alt="snail shell icon"
                      height="40"
                      src="snail-shell.svg"
                      width="40"
/>,
                    "nestedItems": Array [],
                    "nestedLevel": 0,
                    "onKeyboardFocus": [Function],
                    "onMouseEnter": [Function],
                    "onMouseLeave": [Function],
                    "onNestedListToggle": [Function],
                    "onTouchEnd": [Function],
                    "onTouchStart": [Function],
                    "open": null,
                    "primaryText": "anxiety",
                    "primaryTogglesNestedList": false,
                    "secondaryTextLines": 1,
                    "style": Object {
                      "color": "#2F9C95",
                      "fontSize": "18px",
                      "fontWeight": "bold",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "disabled": false,
                "fullWidth": false,
                "label": "Edit Profile",
                "labelPosition": "after",
                "onClick": [Function],
                "primary": false,
                "secondary": false,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
